lib/api-client.js

```diff
@@ -33,19 +32,25 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
       } else {
         errorMessage = 'Unknown error (bad response body)';
       }
-    } else if (response.text.indexOf('<!DOCTYPE') !== -1) {
+    } else if (error.response.text.indexOf('<!DOCTYPE') !== -1) {
       // Manually set a reasonable error when we get HTML back (currently 500s will do this).
```

Comment:
If we have the status code in the response, why are we sniffing the response body to test for a 500 error?